zeppelin-web/src/app/notebook/paragraph/result/result.controller.js

```diff
@@ -769,6 +769,9 @@ function ResultCtrl ($scope, $rootScope, $route, $window, $routeParams, $locatio
       for (let index in row) {
         let stringValue = (row[index]).toString()
         if (stringValue.indexOf(delimiter) > -1) {
+          if (stringValue.indexOf('\"')) {
+            stringValue = stringValue.replace('\"', '\"\"')
```

Comment:
I think the way to escape `"` is `\"`?
actually according to wikipedia both approaches might work, depending on the preference of the app

Reply:
From above PR test description, the comma separator is inside the quote, this csv case is ok.

When the string value is **fo"o \t b"ar**, then it goes to tsv case. 
The "\t" in **fo"o \t b"ar** can be correctly read by Excel. But if the string value is **fo"o	b"ar**, in which contains "\t" typed through **keyboard**, this string value will be misunderstand by Excel. So we still need to deal with tsv case.